resources/assets/js/home.js

```diff
@@ -521,7 +526,7 @@ window.PoliceScorecardHome = window.PoliceScorecardHome || (function () {
                         layers: ['county-join']
                     });
 
-                    if (!features.length) {
+                    if (!features.length || typeof features[0].state === 'undefined' || typeof features[0].state.url === 'undefined') {
```

Comment:
Extract into variable for clarity?

```suggestion
                    const hasNoCountyData = !features.length || typeof features[0].state === 'undefined' || typeof features[0].state.url === 'undefined';
                    if (hasNoCountyData) {
```

Reply:
I am fine with this, but I personally do not like making one time use variables, especially since it causes memory bloat.  Could just be a comment above the conditional IMO if we are looking to avoid confusion.  Thoughts?

Reply:
The memory will be collected once out of scope.